app/models/katello/sync_plan.rb

```diff
@@ -77,6 +81,7 @@ def next_sync_date
         seconds = self.sync_date.sec - now.sec
 
         next_sync = nil
+        now_time = Time.at(now.hour * 60 * 60 + now.min * 60 + now.sec)
```

Comment:
i think using `self.sync_date.utc.strftime( "%H%M%S%N" )` is a better method than `Time.at(now.hour * 60 * 60 + now.min * 60 + now.sec)`, as that seems very manual